upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -1132,7 +1132,7 @@ func (b *BootstrapChannelBuilder) buildAddons() *channelsapi.Addons {
 
 	if b.cluster.Spec.Networking.Cilium != nil {
 		key := "networking.cilium.io"
-		version := "1.6.4-kops.3"
+		version := "1.6.7-kops.1"
```

Comment:
We're not actually changing the Cilium template, so this bump seems unnecessary.

Reply:
Seems better to be consistent. I don't really like moving the version outside the template.

Reply:
I prefer moving the version outside of the template as it allows for easier rollback. But I set the version of Cilium to 99.99.99 to prevent kops from trying to manage it.

Reply:
I see we have now embedded the "expected" version in the manifest, which is more how we've done it.

I'm very open to discussing how we do the management of these versions (and indeed I'm hoping to get addon management to a much better place generally), but this version bump is in keeping with how we've done it to date (and doesn't seem to break your workaround @johngmyers  - we have a few more years yet!), so I'm going to approve.